sdk/iOS/src/MSManagedObjectObserver.h

```diff
@@ -14,4 +17,7 @@
 
 - (instancetype) initWithClient:(MSClient *)client;
 
+/// Block to be called on each operation that will is inserted into MS_TableOperations
```

Comment:
Can you add a note here that not handling errors in this block can result in a mismatch between your client and your server.  I think it will be easy for someone to think of this as optional  step and get their apps in a bad state.  

Long term I'd like to get some storage mechanism that ensures the state has been resolved adequately, but for now that would be good enough.